src/Message/CategoryFilter.php

```diff
@@ -18,7 +18,7 @@
  *
  * @internal
  */
-final class MessageCategoryFilter
+final class CategoryFilter
```

Comment:
Seems like `getExcluded()` and `getIncluded()` are used in tests only. Likely they should not be exposed at all.